CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+# Contributing
```

Comment:
Would like to update this file to something along the lines of https://gist.github.com/PurpleBooth/b24679402957c63ec426